app/grandchallenge/evaluation/templates/evaluation/leaderboard_detail.html

```diff
@@ -13,101 +13,114 @@
     </ol>
 {% endblock %}
 
+{% block sidebar %}
+    <div class="nav nav-pills col-12 pl-3 mt-3">
+        {% for phase in challenge.phase_set.all %}
+            <li class="nav-item">
+                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:leaderboard' and request.resolver_match.kwargs.slug == phase.slug %}active{% endif %}"
+                   href="{% url 'evaluation:leaderboard' challenge_short_name=challenge.short_name slug=phase.slug %}">{{ phase.title }}</a>
+            </li>
+        {% endfor %}
+    </div>
+{% endblock %}
+
 {% block content %}
-    <h2>{{ phase.title }} Leaderboard {{ request.GET.date }}</h2>
+    <div class="mt-3">
```

Comment:
Why has this whole block been wrapped in a new div?

Reply:
So that I could add a custom margin to this block to make it align nicely with the nav-pills / nav-tabs. The alternative, as far as I understand, would have been to add the margin to `base.html` but I thought it would be better not to mess with `base.html` since other templates inherit from there. Not sure though.

Reply:
Why not change `<div class="nav nav-pills col-12 pl-3 mt-3">` to `<div class="nav nav-pills col-12 pl-3 my-3">`? Where `my-3` is shorthand for `mt-3 mb-3`.

Reply:
yes, but that would not affect the content block, right? and it's the content block that was not nicely aligned and that I added the additional `<div class="mt-3">` to

Reply:
oh, sorry, I get what you mean now, sorry! will make the change! ;-)